build/Dockerfile

```diff
@@ -1,10 +1,38 @@
 FROM debian:jessie
 MAINTAINER Thomas Deinhamer <thasmo@gmail.com>
 
+# Install dependencies.
+RUN dpkg --add-architecture i386 \
+	&& apt-get update -y \
+	&& apt-get install -y --no-install-recommends ca-certificates curl lib32gcc1 lib32stdc++6 libcurl4-gnutls-dev:i386 \
+	&& apt-get clean \
+	&& rm -rf /var/lib/apt/lists/* \
+  && gpg --keyserver pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
+  && curl -o /usr/local/bin/gosu -fSL "https://github.com/tianon/gosu/releases/download/1.7/gosu-$(dpkg --print-architecture)" \
+	&& curl -o /usr/local/bin/gosu.asc -fSL "https://github.com/tianon/gosu/releases/download/1.7/gosu-$(dpkg --print-architecture).asc" \
+	&& gpg --verify /usr/local/bin/gosu.asc \
+	&& rm /usr/local/bin/gosu.asc \
+	&& chmod +x /usr/local/bin/gosu
+
+# Add steam user.
+RUN adduser --home /var/lib/steam --disabled-login --gecos "Steam user" steam
+
+# Install SteamCMD.
+RUN gosu steam mkdir -p /var/lib/steam/steamcmd/ \
+	&& curl -SL "http://media.steampowered.com/installer/steamcmd_linux.tar.gz" | gosu steam tar -xzvC /var/lib/steam/steamcmd \
+	&& chmod 744 /var/lib/steam/steamcmd/steamcmd.sh \
+	&& gosu steam /var/lib/steam/steamcmd/steamcmd.sh \
+		+@ShutdownOnFailedCommand 1 \
```

Comment:
I want to split layers. Wit this first steamcmd.sh I download all the files that steamcmd downloads when you run it the first time. This way when the second only downloads the game. If the game change but not steam we don't need to download steam because it's in a previus layer.